### PR TITLE
Ladder command added

### PR DESCRIPTION
**Pull request by padster**

### CODE REVIEW

File: command/ladder.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+            to: channelID,
+            message: "A single playlist must be provided."
+        });
+        return;
+    }
+    var rankedUsers = [];
+    var playlist = playlists[0];
+    // First step, load all the cached results.
+    // Note: This doesn't fetch new ranks, so they may be stale.
+    db.Season.find(function (err, users) {
```
Comment (prayansh) [inline command/ladder.js]:
```javascript
getPlayerDataBatch = function (playerData, callback) {
    console.log("Getting Player Data");
    let Request = unirest.post('https://api.rocketleaguestats.com/v1/player/batch');

    Request.headers({
        'Authorization': getAuth('rl'),
        'Accept': 'application/json',
        'Content-Type': 'application/json'
    });
    Request.send(playerData);

    Request.end(function (response) {
        if (response.status === 429) {
            console.log("Request has been ratelimited: '" + response.body.message + "'.");
        }

        callback(response.status, response.body);
    });
};```

Comment (prayansh) [reply command/ladder.js]: function to get batch player data

Comment (prayansh) [reply command/ladder.js]:
```javascript
let queryPayload = [];
memberIds.forEach(function (id) {
    queryPayload.push({discordId: id});
});
let query = {'$or': queryPayload};
User.find(query, function (err, users) {
    if (users) {
        let usermap = {};
        let batchPayload = [];
        users.forEach(function (user) {
            batchPayload.push({"platformId": user.platform, "uniqueId": user.steamId});
            usermap.steamId = user.discordId;
        });
        getPlayerDataBatch(batchPayload, function (status, body) {
            console.log(`
PlayerDataBatch: responseCode = ${status}`);
            console.log('PlayerDataBatch body=' + JSON.stringify(body));
            let teams = formTeams(body, [[], []]);
            let team1 = teams[0];
            let team2 = teams[2];
        });
    }
});

**Comment by padster** [comment]
Updated, now doing a batch query to get the latest ranks (not saving locally though)

**Review by prayansh — APPROVED** [review]

**Pull request merged**